osu.Game/Overlays/Direct/DirectPanel.cs

```diff
@@ -44,10 +44,13 @@ public abstract class DirectPanel : Container
 
         protected override Container<Drawable> Content => content;
 
-        protected DirectPanel(BeatmapSetInfo setInfo)
+        private readonly bool fetchOnline;
+
+        protected DirectPanel(BeatmapSetInfo setInfo, bool fetchOnline = false)
```

Comment:
Any reason this is `false` by default whereas `DirectGridPanel` has `true` by default?